nntrainer/tensor/tensor.cpp

```diff
@@ -9,6 +9,7 @@
  * @bug		No known bugs except for NYI items
  */
 
+#include <bcq_tensor.h>
```

Comment:
Doesn't it need `ENABLE_BIQGEMM` ?
```suggestion
#ifdef ENABLE_BIQGEMM
#include <bcq_tensor.h>
#endif
```

Reply:
You're right! I'll fix it right away